webserver/webserver/src/main/java/io/helidon/webserver/ForwardingHandler.java

```diff
@@ -355,6 +366,15 @@ protected void channelRead0(ChannelHandlerContext ctx, Object msg) {
      */
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
+        LOGGER.fine(() -> log("Exception caught: %s", ctx, cause.toString()));
+
+        // We ignore stream resets (RST_STREAM) from HTTP/2
+        if (cause instanceof Http2Exception.StreamException
+                && ((Http2Exception.StreamException) cause).error() == Http2Error.CANCEL) {
+            return;     // no action
```

Comment:
no action, including `ctx.close()`?

Reply:
Right we do not want close the connection in this case